ibis/backends/bigquery/compiler.py

```diff
@@ -338,7 +340,11 @@ def _formatter(translator, expr):
 }
 
 
-_operation_registry = impala_compiler._operation_registry.copy()
+# _operation_registry = impala_compiler._operation_registry.copy()
+_operation_registry = {
+    **impala_compiler._operation_registry.copy(),
+    **base_sql._operation_registry.copy(),
```

Comment:
I'd assume that constructing a dictionary this way will make a copy, so the `.copy()` are probably not needed. If you can do a quick test to make sure that would be great.

Better to have `base_sql` first, and `impala` later (see comment below).

Also, you can delete the comment.

Reply:
Sure.  I think you are right about the copy.  I did the following:

```
>>> a = {"A": 1, "B": 2}
>>> b = {**a}
>>> a is b
False
```

Is that a good test?

Reply:
Yes, thanks, safe to remove the `.copy()`.